torch/onnx/_internal/onnxruntime.py

```diff
@@ -201,51 +196,6 @@ def _move_placeholder_to_front(graph_module: torch.fx.GraphModule) -> None:
         first_not_placeholder.prepend(placeholder)
```

Comment:
I have a kind of `unrelated Q`. I have also seen `get_attr` node comes before `placeholder` node, and that break our sequential conversion processsing in `fx_onnx_interpreter.py`, and you have a pass under `virtualization` to fix that. Based on your understanding, Is "get_attr should come before placeholder" a criteria of fx graph? Because I think if it is, PyTorch should maintain it.